src/main/java/com/cdancy/jenkins/rest/domain/crumb/Crumb.java

```diff
@@ -24,18 +24,34 @@
 
 import com.cdancy.jenkins.rest.domain.common.Error;
 import com.cdancy.jenkins.rest.domain.common.ErrorsHolder;
-import com.cdancy.jenkins.rest.domain.common.Value;
 import com.cdancy.jenkins.rest.JenkinsUtils;
 import com.google.auto.value.AutoValue;
 
 @AutoValue
-public abstract class Crumb implements Value<String>, ErrorsHolder {
+public abstract class Crumb implements ErrorsHolder {
+
+    @Nullable
+    public abstract String value();
+
+    @Nullable
+    public abstract String sessionIdCookie();
 
     @SerializedNames({ "value", "errors" })
-    public static Crumb create(@Nullable final String value,
+    public static Crumb create(final String value,
+            final List<Error> errors) {
+
+        return create(value, null, errors);
+    }
+
+    @SerializedNames({ "value", "sessionIdCookie" })
+    public static Crumb create(final String value, final String sessionIdCookie) {
+        return create(value, sessionIdCookie, null);
+    }
```

Comment:
With these changes I _believe_ this `create` method would no longer be used if I'm not mistaken. If that's so feel free to rip it out and put the `@SerializedNames` anno on the new method.

Reply:
Sorry, I don't understand which `create` method you have in mind. The new private one with 3 arguments?
I don't know what is the purpose of the `SerializedNames` annotation, I guessed it has to be put on public methods, I just mimic the previously existing public method. But I can remove the anno from the 2 public methods and put it only on the private one.
Do you think running integration tests is enough to ensure this change make all things work?

Reply:
It was only a possible suggestion. If your IDE says it's no longer being used you can yank it out otherwise just leave it. No worries one way or another.

Reply:
ok
All public methods of this class are used